### DISCUSSION TITLE
Possibility of using BibBot on iPhone

### DISCUSSION THREAD

**Discussion by floriankoenig1** [stefanw/voebbot#437] [Q&A]
Hi,

I am trying to get the bot running on my iPhone since I read most of the articles there. However it is not possible to install custom add-ons for Safari on the phone.
I managed to install the bot by using the Orion browser. The bot is also running, but is not able to open the press database and no additional Genios tab is opened. I know that it is not intended to run on the phone, but it would be great. Any further ideas on how to achieve this?

Thank you in advance!

Here is an example screenshot of the test page stuck in the search for the database:

![IMG_8123](https://github.com/stefanw/bibbot/assets/158174307/acef62f9-6b04-4b48-a62d-c1a2cad26fb9)

**Comment by stefanw** [comment]
BibBot can be run as an extension in Mobile Safari and works but BibBot is not on the iOS App Store. Any idea how to debug extensions in the Orion browser?

**Reply by floriankoenig1** [reply]
Thats good to know. Yes since it is not available in the iOS store its currently impossible to get it running on the phone in Safari. The only way to install it somehow on iPhone was the Orion browser. I am not yet familiar with it, but maybe this is a good start for debugging:

https://help.kagi.com/orion/support-and-community/troubleshooting/troubleshooting-extension-issues.html

**Comment by Netswipe** [comment]
Is there anything new at this point? Any way of using bibbot on iPhone or iPad? Thanks

**Comment by stefanw** [comment]
BibBot was rejected by the current App Store policies. It might be allowed via alternative stores available for EU citizens, but that option still needs to be explored.

If you have an Apple Developer account you can run convert the web extension to a Safari web extension and deploy it on your own devices:
https://developer.apple.com/documentation/SafariServices/safari-web-extensions
